### PR TITLE
Added multibyte slice! example to doc [ci skip]

### PR DESCRIPTION
**Pull request by gouravtiwari**
I added an example of what is expected output from slice! method. 

I also checked and saw that this method rightly returns `Exception` and not `nil`, but document says it returns `nil`. I will write a failing test and correct the document too. 
Quick question, shall I write the test in this PR or in separate PR?

### CODE REVIEW

File: activesupport/lib/active_support/multibyte/chars.rb [hunk 1]
```diff
@@ -88,6 +88,12 @@ def split(*args)
 
       # Works like like <tt>String#slice!</tt>, but returns an instance of
       # Chars, or nil if the string was not modified.
+      #
+      #   string = 'Welcome'
+      #   string.mb_chars.slice!(3)    # => #<ActiveSupport::Multibyte::Chars:0x000000038109b8 @wrapped_string="c">
+      #   string # => 'Welome
```
Comment (claudiob) [inline activesupport/lib/active_support/multibyte/chars.rb]: @gouravtiwari I believe there is a `'` missing here.

**Comment by claudiob** [comment]
If this PR is just a `[ci skip]` about docs, then you should have the other test/docs in a separate PR, otherwise the tests won't be running on Travis CI.

File: activesupport/lib/active_support/multibyte/chars.rb (hunk 1 above)
Comment (gouravtiwari) [inline activesupport/lib/active_support/multibyte/chars.rb]: oops typo! I will fix it and squash my changes.

**Comment by gouravtiwari** [comment]
@claudiob In this PR I only want to update the example. I will report issue with tests/docs in another PR. I squashed my changes into one commit.

**Pull request merged**